internal/cloud/backend_runTasks.go

```diff
@@ -138,12 +181,150 @@ func (b *Cloud) runTasksWithTaskResults(context *IntegrationContext, output Inte
 	})
 }
 
-func (b *Cloud) runTasks(ctx *IntegrationContext, output IntegrationOutputWriter, stageID string) error {
-	return b.runTasksWithTaskResults(ctx, output, func(b *Cloud, stopCtx context.Context) (*tfe.TaskStage, error) {
+func (b *Cloud) runTaskStage(ctx *IntegrationContext, output IntegrationOutputWriter, stageID string) error {
+	err1 := b.runTasksWithTaskResults(ctx, output, func(b *Cloud, stopCtx context.Context) (*tfe.TaskStage, error) {
 		options := tfe.TaskStageReadOptions{
-			Include: []tfe.TaskStageIncludeOpt{tfe.TaskStageTaskResults},
+			Include: []tfe.TaskStageIncludeOpt{tfe.TaskStageTaskResults, tfe.PolicyEvaluationsTaskResults},
 		}
 
 		return b.client.TaskStages.Read(ctx.StopContext, stageID, &options)
 	})
+	// call to runTaskwithPolicyEvaluation
+	err2 := b.runTasksWithPolicyEvaluation(ctx, output, func(b *Cloud, stopCtx context.Context) (*tfe.TaskStage, error) {
+		options := tfe.TaskStageReadOptions{
+			Include: []tfe.TaskStageIncludeOpt{tfe.TaskStageTaskResults, tfe.PolicyEvaluationsTaskResults},
+		}
+
+		return b.client.TaskStages.Read(ctx.StopContext, stageID, &options)
+	})
+	//TODO: clean this up
+	if err1 != nil && err2 != nil {
+		return fmt.Errorf("error running task stage: %v %v", err1, err2)
+	} else if err1 != nil {
+		return err1
+	} else if err2 != nil {
+		return err2
+	}
+	return nil
+
+}
+
+func (b *Cloud) runTasksWithPolicyEvaluation(context *IntegrationContext, output IntegrationOutputWriter, fetchTaskStage taskStageReadFunc) error {
```

Comment:
probably wouldn't call this `runTasks....`  perhaps?
```suggestion
func (b *Cloud) taskStageWithPolicyEvaluation(context *IntegrationContext, output IntegrationOutputWriter, fetchTaskStage taskStageReadFunc) error {
```